### ISSUE TITLE
Semantic router does not work with latest versions of Cohere

### ISSUE THREAD

**Issue by marygriffus** [aurelio-labs/semantic-router#439]
When I upgraded to the latest Semantic Router version in my project, I got the following error:

```
  File "/src/app/src/main.py", line 10, in <module>
    from semantic_router.encoders import HuggingFaceEncoder
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/src/app/.venv/lib/python3.11/site-packages/semantic_router/__init__.py", line 1, in <module>
    from semantic_router.hybrid_layer import HybridRouteLayer
  File "/src/app/.venv/lib/python3.11/site-packages/semantic_router/hybrid_layer.py", line 6, in <module>
    from semantic_router.encoders import (
  File "/src/app/.venv/lib/python3.11/site-packages/semantic_router/encoders/__init__.py", line 7, in <module>
    from semantic_router.encoders.cohere import CohereEncoder
  File "/src/app/.venv/lib/python3.11/site-packages/semantic_router/encoders/cohere.py", line 5, in <module>
    from cohere.types.embed_response import EmbedResponse_EmbeddingsByType
ImportError: cannot import name 'EmbedResponse_EmbeddingsByType' from 'cohere.types.embed_response' (/src/app/.venv/lib/python3.11/site-packages/cohere/types/embed_response.py)
```

When I looked at `cohere.types.embed_response` in cohere 5.11.0, there is no `EmbedResponse_EmbeddingsByType`- it seems to have been one-to-one replaced by `EmbeddingsByTypeEmbedResponse`.

**Comment by jamescalam** [comment]
@marygriffus this should be resolved with SR 0.0.68, we saw this issue initially with #422 and made the changes. Could you confirm you have `0.0.68`?

**Comment by marygriffus** [comment]
Apologies, was on `0.0.66`, can close this!
